### DISCUSSION TITLE
Two App Suggestions

### DISCUSSION THREAD

**Discussion by ghost** [Divested-Mobile/DivestOS-Build#365] [General]
Hey,

i saw that Aperture Camera and LineageOS Clock come as pre installed and I wanted to suggest two better Alternatives in my Opinion.

Instead of Aperture Camera -> Secure Camera from GrapheneOS. It will receive an overhaul soon and it will be the first App that get's an overhaul, other than the AOSP Apps, which will get after that.

Instead of LineageOS Clock -> NFC Alarm Clock. The Benefits: It is active maintained/updated and will get Direct Boot next Month. It will probably get Material You Design soon too and it is even on IzzyOnDroid Repo. And it already has Material Icon to match all Apps too. And it will probably come to Accrescent in the Future too. I think this Clock App has lot of Benefits, because the Developer of this App is open to implement good things and is fast by fixing/adding too. Here it is: https://github.com/gabeg805/NFC-Alarm-Clock/releases

I hope you can take them as pre installed Apps, because they have more Benefits, specially Direct Boot is important. I don't mean that it should get directly implemented, rather I mean that after the Clock App has got Direct Boot. But secure Camera could be implemented.

**Comment by SkewedZeppelin** [comment]
Aperture is included specifically because it hardcodes support for the extra cameras on the supported devices.
SecureCamera can only support the extra cameras if the vendor correctly set them up, which most do not.


Replacing the clock isn't beneficial enough to go through the arduous user migration process.

**Reply by ghost** [reply]
Okay, I understand both of your points.

**Reply by ghost** [reply]
I found another even better Alternative than that. Which even has Material You Design too. And it has Direct Boot and Material Icon too. It is based on AOSP and Privacy-conscious. And from the Developer of HeliBoard. Here: https://github.com/BlackyHawky/Clock
